docs/application/web/get-started/mobile/first-app.md

```diff
@@ -228,8 +226,7 @@ To run the application on the emulator:
 
         ![Application running in the emulator](media/emulator_running_mw.png)
 
-        > **Note**
-        >
+        > [!NOTE]
         > If the emulator display has switched off, you cannot see the application launch. To see the application on the emulator screen:
```

Comment:
It looks like the list items in the note are in green, the same color with the title, "Note".
Is it intended?

![image](https://user-images.githubusercontent.com/9958374/71966902-c0fa9000-3245-11ea-9250-80580a431951.png)